docs/usermanual/index.md

```diff
@@ -17,16 +17,159 @@ TODO
+                                  dst_profile='analysis_profile')
+```
+
+The equivalent CLI command is:
+
+```bash
+# Create a forensic copy of the volume 'vol1' from one account to another 
+libcloudforensics aws 'us-east-2b' copydisk --volume_id='vol1' --src_profile='src_profile' --dst_profile='dst_profile' 
+```
+
+If you want to make the copy in a different zone, just specify `dst_zone` in the above method.
+You can also pass a dictionary of tags to add to the volume copy.
+
+```python
+from libcloudforensics.providers.aws import forensics
+
+# Create a forensic copy of the volume 'vol1' from one account to another, in a different zone
+
+copy = forensics.CreateVolumeCopy('us-east-2b',
+                                  dst_zone='us-west-2a',
+                                  volume_id='vol1', 
+                                  src_profile='src_profile', 
+                                  dst_profile='analysis_profile',
+                                  tags={'customTag': 'customValue'})
+```
+
+The equivalent CLI command is:
+
+```bash
+# Create a forensic copy of the volume 'vol1' from one account to another, in a different zone
+libcloudforensics aws 'us-east-2b' copydisk --dst_zone='us-west-2a' --volume_id='vol1' --src_profile='src_profile' --dst_profile='dst_profile' --tags="{'customTag': 'customValue'}" 
+```
+
+Finally, you can opt to pass an `instance_id` parameter to the method instead of a `volume_id`.
+If you do so, then the boot volume of the instance pointed to by `instance_id` will be copied.
+
+```python
+from libcloudforensics.providers.aws import forensics
+
+# Create a forensic copy of the boot volume of instance 'inst1' in the default account.
+
+copy = forensics.CreateVolumeCopy('us-east-2b',
+                                  instance_id='inst1')
+```
+
+The equivalent CLI command is:
+
+```bash
+# Create a forensic copy of the boot volume of instance 'inst1' in the default account
+libcloudforensics aws 'us-east-2b' copydisk --instance_id='inst1'
+```    
 
 #### EBS encryption
 
-TODO
+Making copies of encrypted resources is a bit more involved. By default, encrypted Elastic Block Store (EBS) resources 
+use the aws/ebs key that is associated with the AWS account. Snapshots that are encrypted with this key cannot be shared 
+directly. Instead, you must first make a copy of the snapshot, and re-encrypt it with a Customer Managed Key (CMK) key. 
+Both the CMK key and the snapshot copy need to then be shared with the destination account.
+
+
+To reduce key management hassle, libcloudforensics allows you to transfer encrypted EBS resources between accounts by 
+generating a one-time use CMK key. This key is deleted once the process completes. The process is depicted below:
+
+![ebs-encryption](https://lh4.googleusercontent.com/U_DxaWaZqx00gSG8kfhlwB6dkGGcPi5_Cd_qgZR9VMdW0rWmrHD-VDeChMKNBfJJ86qBJC9duaZzHbmBxWU1WpsNkNYmp3UQvmU2R9EuDmn0vJJqy0rZVTjZVBePIXvb8QlJmBgq)
```

Comment:
I encourage you to host this image within this git repo

Reply:
Added the image to the repo. Note that the image will be live only once this is merged to master.